.github/workflows/live-testnet-tests.yml

```diff
@@ -0,0 +1,362 @@
+name: Live Testnet Tests
+on:
+  schedule:
+    - cron: "0 0 * * *" # Run nightly
+  push:
+    tags:
+      - "*"
+  workflow_dispatch:
+
+env:
+  CHAINLINK_IMAGE: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}.dkr.ecr.${{ secrets.QA_AWS_REGION }}.amazonaws.com/chainlink
+  INTERNAL_DOCKER_REPO: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}.dkr.ecr.${{ secrets.QA_AWS_REGION }}.amazonaws.com
+  MOD_CACHE_VERSION: 2
+
+  CHAINLINK_COMMIT_SHA: ${{ github.sha }}
+  CHAINLINK_ENV_USER: ${{ github.actor }}
+  TEST_LOG_LEVEL: debug
+  EVM_KEYS: ${{ secrets.QA_EVM_KEYS }}
+
+  OPTIMISM_GOERLI_URLS: ${{ secrets.QA_OPTIMISM_GOERLI_URLS }}
+  OPTIMISM_GOERLI_HTTP_URLS: ${{ secrets.QA_OPTIMISM_GOERLI_HTTP_URLS }}
+
+  ARBITRUM_GOERLI_URLS: ${{ secrets.QA_ARBITRUM_GOERLI_URLS }}
+  ARBITRUM_GOERLI_HTTP_URLS: ${{ secrets.QA_ARBITRUM_GOERLI_HTTP_URLS }}
+
+jobs:
+  build-chainlink:
```

Comment:
I think this is duplicated between the two workflows now. We need to either move it to an action now or make a ticket to do so in the near future so we don't have to update multiple places.

Reply:
Definitely duplicated, I can move it into a shared action. My other concern with it (which I wanted to test out after merge, but maybe you know) is that often, both this and the simulated workflow will run in tandem. They're going to duplicate effort on this step, building the same image, which is annoying but can be lived with if need be. The other fear I had: will they somehow conflict with each other? 1 builds the image first and uploads, then the second tries to upload but errors?

Reply:
You can add -live to the name of the image pushed if it is an issue. I need to look into if the logic to use a builder workflow that these would wait on works better now than it did a couple years ago when I last played with it. We have a third build that happens as well so it would be nice to bring them all together. I will make a ticket for this.